src/components/Jam.jsx

```diff
@@ -31,6 +39,61 @@ const ValuesListener = ({ handler }) => {
   return null
 }
 
+const DEFAULT_PRECONDITION_SUMMARY = {
+  isFulfilled: false,
+  numberOfMissingUtxos: SCHEDULER_PRECONDITIONS.MIN_NUMBER_OF_UTXOS,
+  amountOfMissingConfirmations: SCHEDULER_PRECONDITIONS.MIN_CONFIRMATIONS_OF_SINGLE_UTXO,
+  amountOfMissingOverallRetries: SCHEDULER_PRECONDITIONS.MIN_OVERALL_REMAINING_RETRIES,
```

Comment:
How do these retires work exactly? I was under the impression each transaction is a "try". Now it seems like tries are attached to UTXOs. Does that mean if I do a transaction with 5 UTXO, I need 5 tries? Does every commitment give every other UTXO in the same mixdepth 3 tries?

Reply:
This is controlled by config param `taker_utxo_retries`. By default, this is set to `3`. 

> Now it seems like tries are attached to UTXOs

Yes, this is per utxo. The `/utxo` response from the API has following properties included:
```ts
  tries: number
  tries_remaining: number
```
With an untouched/unused utxo it will return:
```ts
{
  [...]
  "tries": 0,
  "tries_remaining": 3
}
```
> Does that mean if I do a transaction with 5 UTXO, I need 5 tries?

No, I do not think so. I think you need 1 commitment and will have 5 * 3 = 15 tries for that. Not 100% sure.

> Once you've tried 3 times, you have to use a different utxo (some Makers may choose to be more lax, but this will be the usual requirement)`

It is not _totally_ clear how it all works from https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/SOURCING-COMMITMENTS.md#commitments-for-initiating-coinjoins, but I think the assumption that you need at least one remaining try, like implemented in the code, is valid and a minimum requirement. I'd be in favor of keeping it.

> As an example, if you send to 3 addresses instead of 1, you will then have 9 commiments available - effectively 9 tries at doing the first transaction, instead of only 3 tries if you send the coins to one address.